packages/text-field/icon/README.md

```diff
@@ -0,0 +1,17 @@
+# React Text Field Icon
+
+MDC React Text Field Icon is a React Component which uses MDC [MDC Text Field Icon](https://github.com/material-components/material-components-web/tree/master/packages/mdc-textfield/icon/)'s CSS and foundation JavaScript.
+
+## Props
+
+Prop Name | Type | Description
+--- | --- | ---
+disabled | Boolean | Toggles the disabled state of the icon.
+children | Element | Required. Expects a single child icon element.
+
```

Comment:
Add a bit about "Integrating Component"

like you did here:
https://github.com/material-components/material-components-web-react/tree/master/packages/top-app-bar#navigation-icon